src/status_im/utils/ethereum/core.cljs

```diff
@@ -80,17 +87,26 @@
   (if b "0x0" "0x1"))
 
 (defn hex->int [s]
-  (js/parseInt s 16))
+  (if (= s hex-prefix)
+    0
+    (js/parseInt s 16)))
 
 (defn int->hex [i]
   (.toHex dependencies/Web3.prototype i))
 
 (defn hex->bignumber [s]
   (money/bignumber (if (= s hex-prefix) 0 s)))
 
+(defn hex->address [s]
+  (when (= 66 (count s))
```

Comment:
66 ? :fearful: Can you add a comment here?

Reply:
doesn't everybody know what order 66 is ?